tools/getkeys/index.js

```diff
@@ -16,8 +16,9 @@ const appendFile = util.promisify(fs.appendFile);
 
 // Wraps the given string in quotes, escaping any quotes already present in the string
 // with '\"', which is compatible with cmd, bash, and zsh.
+// Also, backticks in the middle of a string break the string and cause shells to complain about unmatched " characters.
 function quote(str) {
-	return `"${str.split('"').join('\\"')}"`;
+	return `"${str.split('"').join('\\"').replace('`', '\\`')}"`;
```

Comment:
what happens with this value? will all shell need this? i don't think windows shell needs ` escaped, and the escaping will break it.

Reply:
We only call this for `bash` and `zsh`, and VSCode's parsing of `.zshrc` and `.bashrc` suggests they both need the fix.

Windows saves the raw values in the registry and doesn't do any escaping so this doesn't change the Windows experience. For `fish` we're not escaping anything today either; it does something different than `bash` and `zsh`, so if it's working this won't break it, and if it's broken already this just won't fix that particular shell.

Reply:
Probably worth adding these details to the comments.

Reply:
On adding the comments I realized we _are_ calling the function on Windows; checking that case.

Reply:
Should be addressed now.